dipy/workflows/analysis.py

```diff
@@ -0,0 +1,377 @@
+from __future__ import division, print_function, absolute_import
+
+import logging
+from dipy.workflows.workflow import Workflow
+import numpy as np
+from dipy.io.image import load_nifti
+from scipy.spatial import cKDTree
+from scipy.ndimage.interpolation import map_coordinates
+from dipy.io.streamline import load_trk
+from dipy.tracking.streamline import transform_streamlines
+from dipy.tracking.streamline import set_number_of_points
+from dipy.tracking.streamline import Streamlines
+from dipy.segment.clustering import QuickBundles
+from dipy.segment.metric import AveragePointwiseEuclideanMetric
+import os
+import pandas as pd
+from scipy import spatial
+from dipy.io.peaks import load_peaks
+import statsmodels.formula.api as smf
+import matplotlib.pyplot as plt
+
+
+def plot(x, y, title, file_name):
+
+    """ saves the simple plot with given x and y values
+
+    """
+
+    plt.plot(x, y)
+    plt.xlabel('disk no')
+
+    plt.ylabel("-log10(pvalues)")
+
+    plt.title(title)
+
+    plt.savefig(file_name)
+
+    plt.clf()
+
+
+def save_hdf5(dt, fname):
```

Comment:
Can't we use the pandas [to_hdf](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.to_hdf.html) method instead of writing our own function for this?

Reply:
Fine with me.

Reply:
within the save_hdf5 function, I am using pandas. I have created function just for the re-usability purpose.

Reply:
Yes, it makes sense to have a wrapper at io. With load and save options. If you go that way you should have fname first and then dt. However, if this is not used so often and given the release is very soon I would  suggest going with to_hdf. Fine with either.

Reply:
Hi @arokem,
I tried to_hdf fucntion of pandas but it didn't work for me. It's not very flexible.
Now, I am creating _save_hdf5 internal function.